### ISSUE TITLE
[OUDS] Component Creation - Input Text

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/Orange-Boosted-Bootstrap#2825]
As a user of the Ouds library
I want to be able to use the component
So that I can
- access to the documentation on how to use the component
- see the available options and observe the changes on the component itself
- understand how to set it up based on the designer's customization
- see the code associated to the component customization
- copy and paste the code based on the selected customization
- and add this component in my own projects

## Context
The component is described in the Design tools and allows to describe the various states and variants necessary to build this component.
The component should be working correctly in the Light and Dark mode.
The component should be working fine in the various themes proposed by the library.

## Variants
TBD

## Change log
Take into account the latest version updates (current is at least 1.3.0)
* The name of the “Style” variant has been replaced to “Outlined” with true/false variant
* design token updates to 1.5.0

**Comment by Aniort** [comment]
hello,
Just one thing to correct:
The error messages are inside `div`, it's better to put them inside a semantic tag, such as `p`
That's all, folks!

**Comment by nilloq** [comment]
> Hello [@nilloq](https://github.com/nilloq) [@vprothais](https://github.com/vprothais) Voici la review design Figma : https://www.figma.com/design/yKtkEY1DOl7YDvmt7jq0QF/-OUDS-Core-Lib--Review?node-id=69780-13361&t=D1RwOUG9XZmoihAb-1
> 
> J'ai quelques décalages sur des textes en fonction de la situation. Je les ai commenté sur la review.

Fixed in last release, see my comments on Figma @jerome-regnier
